src/main/java/org/radarcns/management/domain/SourceData.java

```diff
@@ -78,7 +85,7 @@
     @ManyToMany(mappedBy = "sourceData" , fetch = FetchType.EAGER)
     @JsonIgnore
     @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
-    private Set<DeviceType> deviceTypes = new HashSet<>();
+    private Set<SourceType> sourceTypes = new HashSet<>();
```

Comment:
Should this also be many-to-one? Or can a `SourceData` belong to multiple `SourceType`s?

Reply:
No. Should be ManyToOne. Forgot to change. Will do.